src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js

```diff
@@ -99,6 +132,32 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
       .slice(0, 1)
   }
 
+  const logWorkOrders = async (
```

Comment:
This is good info to have and definitely very useful as-is. Could consider adding some timestamps so we won't need to look them up after?

So in progress work orders, started work orders, and current work orders as objects including id, start /end times, and scheduled start / end times.

But if this needs API changes then don't worry about it for now I guess - this is very good as-is

Reply:
timestamps are set within the api

Reply:
We could just log when returning data from the api. But I think whats more useful is knowing what the frontend sees. So we can see the result of any filtering logic on the frontend

Reply:
If we find this useful, there is no reason we couldnt change the implementation